versioned_docs/version-1.3.0-beta/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Multi-Architecture Image/Artifact Support
+sidebar_position: 110
+---
+
+# Multi-Architecture Image/Artifact Support
+
+ORAS can help you create and manage multi-architecture artifacts such as configurations, executables, and container images.
```

Comment:
It's recommeded that add a note at the begining:

> This feature flag `--artifact-platform` of `oras push` is still in experimental. It is subject to change until it is stable.

Reply:
Done